packages/cashier-v2/src/lib/CashierOnboarding/components/CashierOnboardingCryptoCard/CashierOnboardingCryptoCard.tsx

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
+import { useActiveAccount } from '@deriv/api';
 import { onboardingCryptoCardIcons } from '../../constants/icons';
 import { CashierOnboardingCard } from '../CashierOnboardingCard';
 import { CashierOnboardingIconMarquee } from '../CashierOnboardingIconMarquee';
 
-const CashierOnboardingCryptoCard: React.FC = () => {
+type TProps = {
+    setIsDeposit: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const CashierOnboardingCryptoCard: React.FC<TProps> = ({ setIsDeposit }) => {
+    const { data: activeAccount } = useActiveAccount();
+    const isCrypto = Boolean(activeAccount?.currency_config?.is_crypto);
+    const onClickHandler = () => {
+        if (isCrypto) setIsDeposit(true);
+        // eslint-disable-next-line no-alert
+        else alert('Please switch to the crypto account to make a deposit');
```

Comment:
alert? I'm assuming its temporary -might be worth putting TODO or something to make it clear,